src/components/hero/stats-card.js

```diff
@@ -27,21 +27,33 @@ const carouselStyles = css`
   }
 `;
 
+const resourceStyles = css`
+  ${tw`font-light underline`}
+`;
+
+const textStyles = css`
+  ${tw`mx-3 md:mx-6 text-left`}
+`;
+
+const emphasisStyles = css`
+  ${tw`font-semibold`}
+`
+
 const stats = [
   {
-    heading: "24%",
-    text:
-      "People from a working class background earn 24% less a year than someone with a professional background",
+    heading: "51%",
+    text: <span>Of disadvantaged pupils reached the expected standard in all of reading, writing and maths, compared to <span css={emphasisStyles}>71% of all other pupils</span> in 2019.</span>,
```

Comment:
can we rename this to `innerHtml` or something else that indicates it's a JSX Element? also, unless the span is doing something, it could be wrapped with `<>` (I think, haven't checked)

Reply:
oh, also, for accessibility and semantic reasons, I think it's a good idea to use `<strong>` for that "emphasis" span

Reply:
good shout, that name was left over from when this was actually just text 👍